src/core/server/saved_objects/mappings/lib/get_root_properties_objects.ts

```diff
@@ -30,18 +31,20 @@ import { getRootProperties } from './get_root_properties';
  *  This data can be found at `{indexName}.mappings.{typeName}.properties`
  *  in the es indices.get() response where the properties are objects.
  *
- *  @param  {EsMappingsDsl} mappings
- *  @return {EsPropertyMappings}
+ *  @param  {EsMappings} mappings
+ *  @return {EsMappings}
  */
-export function getRootPropertiesObjects(mappings) {
+export function getRootPropertiesObjects(mappings: EsMappings): EsMappings {
```

Comment:
We don't need to define return type mappings, TS knows that this function returns an `EsMappings` object because it's explicitly types at the point of return.